### DISCUSSION TITLE
[Help]: LightBox library for Embla Carousel

### DISCUSSION THREAD

**Discussion by Pixsal** [davidjerleke/embla-carousel#993] [Help] [answered]
### Summary

Any lightbox library you guys recommend that works well with embla? I have created a component for oxygen builder, Integrated embla into it however Need to add lightbox effect when slides are clicked.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by sarussss** [comment, marked as answer]
Hi @Pixsal,
You can try [PhotoSwipe](https://photoswipe.com/)

**Reply by zdenkolini** [reply]
would love a example on the docs for this, seems like it would be a common feature for web shops or similar use cases
